dt/PIDONE/DFG-pidone-18.0-3cont-2comp-ipv4-geneve-HA

```diff
@@ -0,0 +1,43 @@
+RHOS Deployed Topology DFG-pidone-18.0-3cont-2comp-ipv4-geneve-HA
+Based on OpenStack K8S operators from the "main" branch of the OpenStack Operator repo on Oct 17th, 2023
+
+General information:
+Revision	Change		Publication Date
+v0.1	 	Initial		Dec 20 2023
+
+Node topology:
+Node role	                                bm/vm	amount
+Openshift master/worker combo-node cluster	vm	    3
+Compute node					vm	    3
```

Comment:
Hi Julia, Due to the volume's of DT's that have been created we are looking for solutions to consolidate them into Unified DTs so I was hoping to get some clarifications to see if these DT's can be incorporated into some of the larger DT's being proposed in [1].  Does this specific DT require 2 or 3 computes?.


[1] https://github.com/openstack-k8s-operators/architecture/pull/45

Reply:
Hi James. The number of computes doesn't matter.